plugins/module_utils/module_container/base.py

```diff
@@ -938,6 +938,7 @@ def _compare_platform(option, param_value, container_value):
         interval=dict(type='str'),
         timeout=dict(type='str'),
         start_period=dict(type='str'),
+        start_interval=dict(type='str'),
```

Comment:
There also needs to be an API version check for API version >= 1.44 for `start_interval`.

Look for `extra_option_minimal_versions` in plugins/module_utils/module_container/docker_api.py to see similar examples.